vcd/sample_vcd_test_config.json

```diff
@@ -53,7 +53,11 @@
       "//": "Peer definition for a vpn", 
       "peerIp": "192.168.5.1",
       "peerSubnetGw": "192.168.5.1"
-    }
+    },
+    "vcenter" : "vC1",
+    "externalNetwork": "externalNetworkName",
+    "externalNetworkPortGroup": "ForTestingPG",
+    "externalNetworkPortGroupType": "DV_PORTGROUP"
```

Comment:
Which is more correct and generic: "port group" or "vSphere network"?

Reply:
By API is port group

Reply:
OK, port groups they are then.